tools-local/Microsoft.ML.StableApi/Microsoft.ML.StableApi.csproj

```diff
@@ -7,13 +7,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ML" Version="1.0.0" />
-    <PackageReference Include="Microsoft.ML.DataView" Version="1.0.0" />
-    <PackageReference Include="Microsoft.ML.CpuMath" Version="1.0.0" />
-    <PackageReference Include="Microsoft.ML.FastTree" Version="1.0.0" />
-    <PackageReference Include="Microsoft.ML.LightGbm" Version="1.0.0" />
-    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.0.0" />
-    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="1.0.0" />
+    <PackageReference Include="Microsoft.ML" Version="1.1.0" />
```

Comment:
I am curious to know how other teams do it, but are we going to keep increasing the reference packages to the latest GA?

Reply:
I would imagine so. @eerhardt thoughts?

Reply:
That would be my plan. Once you ship `1.0`, you don't want to break any of those APIs. Then you ship new APIs in `1.1`. Now as you are building `1.2` you don't want to break those new APIs you just added in `1.1`.

Reply:
Yes that's what I thought! It makes total sense. Thanks @eerhardt

Reply:
@codemzs If possible we should add this to the notes on how to release a new version.